lib/paperclip/locales/tr.yml

```diff
@@ -0,0 +1,18 @@
+en:
```

Comment:
This should read `tr`, not `en`. Can you please fix and rebase on top of latest master? Thank you for your contribution!